prisma/schema.prisma

```diff
@@ -21,6 +21,11 @@ enum MuscleGroup {
   TRICEPS
 }
 
+enum Gender {
+  MALE
+  FEMALE
```

Comment:
We may add also, BINARY and PREFER_NOT_SPECIFY. WDYT?

Reply:
hahah will do 😈